add Docker compatibility

avanwouwe opened the pull request:
This change ensures that the oAuth mechanism works even inside Docker, by binding to all interfaces and not just the local loopback. In order to do this two dependencies are upgraded. A small explanation is also added to README.md

invictus-korstiaan commented:
Thanks for this PR, really appreciate it. Sorry for not seeing this earlier, we switched to GH organization and mentions weren't configured. I'm getting an error when using Docker with the requirements I modified your command a bit because docker wants a hardcoded path to mount, see screenshots below. 
![Screenshot 2023-12-05 at 17 00 53](https://github.com/invictus-ir/ALFA/assets/21335149/f52c806d-3ee9-497c-9be9-5ba8deecd3b2)

![Screenshot 2023-12-05 at 17 01 12](https://github.com/invictus-ir/ALFA/assets/21335149/4e731f0b-81c2-4fb9-a2fe-8953e8f14ad3)

Now I'm no Docker expert, but before I accept want to make sure I can get it to work.

avanwouwe commented:
Hi. No worries. And of course.

Not sure about the error. It seems to break on a network access when trying
to recover information about the first required package (which I did not
touch)

I was on MacOS, and perhaps Docker on MacOS is more permissive with respect
to internet access. You're on Linux I suppose? Thanks for raising this,
I'll give it a try this weekend on my Linux machine (as I should have
anyway)

On Tue, Dec 5, 2023 at 5:03 PM Invictus-Korstiaan ***@***.***>
wrote:

> Thanks for this PR, really appreciate it. Sorry for not seeing this
> earlier, we switched to GH organization and mentions weren't configured.
> I'm getting an error when using Docker with the requirements I modified
> your command a bit because docker wants a hardcoded path to mount, see
> screenshots below.
> Screenshot.2023-12-05.at.17.00.53.png (view on web)
> <https://github.com/invictus-ir/ALFA/assets/21335149/f52c806d-3ee9-497c-9be9-5ba8deecd3b2>
>
> Screenshot.2023-12-05.at.17.01.12.png (view on web)
> <https://github.com/invictus-ir/ALFA/assets/21335149/4e731f0b-81c2-4fb9-a2fe-8953e8f14ad3>
>
> Now I'm no Docker expert, but before I accept want to make sure I can get
> it to work.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/invictus-ir/ALFA/pull/4#issuecomment-1841094657>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEWM3SC2DGOKJV2XHI4ASVTYH5ATJAVCNFSM6AAAAAA6SVCKPGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTQNBRGA4TINRVG4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

invictus-korstiaan commented:
Hi did you manage to test this, just going through open issues/bugs to see if there's still an action here.

invictus-korstiaan commented:
No updates/activity.
